Source/WTF/wtf/PlatformHave.h

```diff
@@ -688,7 +688,6 @@
 #define HAVE_PASSKIT_APPLE_PAY_LATER_AVAILABILITY 1
 #endif
 
-
```

Comment:
Was this on purpose?

Reply:
Yes, there was an extra new line here so I just removed it as a drive-by fix.